### PR TITLE
fix(invity): coinmarket load invity data if doesn't exist

### PR DESCRIPTION
**Pull request by LukasRada**
Bug repro steps:

1. In coin market Spend tab refresh page.
2. Page stucks in loading state.

Fix:
Just load the data from Invity server in the hook.

### CODE REVIEW

File: packages/suite/src/hooks/wallet/useCoinmarketSpend.ts [hunk 1]
```diff
@@ -42,11 +43,15 @@ const useSpendState = ({ selectedAccount, fees }: Props, currentState: boolean)
...
 
-    const { addNotification, setShowLeaveModal, saveTrade } = useActions({
+    const { addNotification, setShowLeaveModal, saveTrade, loadInvityData } = useActions({
         addNotification: notificationActions.addToast,
         setShowLeaveModal: coinmarketSellActions.setShowLeaveModal,
         saveTrade: coinmarketSpendActions.saveTrade,
+        loadInvityData: coinmarketCommonActions.loadInvityData,
     });
+
+    loadInvityData();
```
Comment (szymonlesisz) [inline packages/suite/src/hooks/wallet/useCoinmarketSpend.ts]: `loadInvityData` will be called every time when `SpendLoaded` will be rendered, every time selectedAccount, device, lang, fees are changed  (i've counted at least 4 times at 1st load)

it should be wrapped in some memo or something, no? 
i noticed that other invity hooks are doing the same thing, calling `@coinmarket-common/load_data` multiple times

Comment (matejkriz) [inline packages/suite/src/hooks/wallet/useCoinmarketSpend.ts]: what about to call it only once not on every re-render?
```suggestion
    useEffect(() => {
      loadInvityData();
    }, [loadInvityData]);
```

Comment (LukasRada) [reply packages/suite/src/hooks/wallet/useCoinmarketSpend.ts]: Fixed.

Comment (LukasRada) [reply packages/suite/src/hooks/wallet/useCoinmarketSpend.ts]: Fixed.

**Review by matejkriz — APPROVED** [review]

**Review by szymonlesisz — APPROVED** [review]

**Pull request merged**